person.rb

```diff
@@ -0,0 +1,32 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'You must implement the correct_name method.'
+  end
+end

```

Comment:
- [ ] [OPTIONAL] As the previous reviewer mentioned it would be nice to make this a separate file to make your codebase readable